docs/dev/agent_integration.md

```diff
@@ -21,6 +33,53 @@ The required Datadog Agent integration development tools include:
 - [Docker][4] to run the full test suite.
 - The git [command-line][5] or [GitHub desktop client][19].
 
+**To get started, select either tab below for instructions on building an out-of-the-box Agent-based integration or an Agent-based integration on the Marketplace.** 
+
+{{< tabs >}}
+{{% tab "Build a Marketplace Integration" %}}
+## Set up a directory and clone the Marketplace repository
+
+Set up a directory:
+
+1. Request access to the [Marketplace repository][101] by following the instructions in the [Marketplace documentation][102].
+2. Create a `dd` directory:
+   {{< code-block lang="shell" >}}mkdir $HOME/dd{{< /code-block >}}
+
+   The Datadog Development Toolkit command expects you to be working in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
+
+3. Once you have been granted access to the Marketplace repository, create the `dd` directory and clone the `marketplace` repo:
+   {{< code-block lang="shell" >}}git clone git@github.com:DataDog/marketplace.git{{< /code-block >}}
+4. Create a feature branch to work in.
```

Comment:
This font looks smaller than for other steps. It also doesn't have a space after it before the next heading. 
![image](https://user-images.githubusercontent.com/116011073/235536280-65a08976-6cfd-4d5e-be89-e1948530db4d.png)